src/main/webapp/chatroom.html

```diff
@@ -6,14 +6,22 @@
     <link rel="stylesheet" href="mapStyle.css">
     <link rel="stylesheet" href="chatStyle.css">
     <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAZdeLtE431GvRd-XmPa9p7BWWev6q94II" type="text/javascript" id="mapAPI"></script>
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
+    <script src="https://www.gstatic.com/firebasejs/7.17.2/firebase-app.js">
+    </script>
+    <!-- TODO: Add SDKs for Firebase products that you want to use
```

Comment:
nit: Attach a user or issue to a TODO. Ex: `TODO(shamadeh): Add SDKs for Firebase products that you want to use
`